packages/kotti-ui/source/kotti-field-inline-edit/KtFieldInlineEdit.vue

```diff
@@ -0,0 +1,339 @@
+<template>
+	<KtField
+		v-bind="{ field }"
+		ref="fieldRef"
+		:class="rootClasses"
+		:getEmptyValue="() => null"
```

Comment:
i think `getEmptyValue` can be removed, i remember it was needed only for `handleClear` and this is not supported by this component